docs/Server-Methods.md

```diff
@@ -169,3 +169,23 @@ fastify.register(function (instance, options, next) {
 #### setErrorHandler
 
 `fastify.setErrorHandler(handler(error, reply))`: set a function that will be called whenever an error happens. The handler is fully encapsulated, so different plugins can set different error handlers, *async await* is supported as well.
+
+<a name="pretty-print"></a>
+#### prettyPrint
```

Comment:
This doesn't really give any clue about what will be printed. How about `printRoutes` instead? Then it could also support:

```js
fastify.printRoutes({pretty: true})
``

With the default being the regular JSON serialized array in a `fastify.log.info` line.

Reply:
I don't think that print the internal tree is a good idea, we store there our marvelous `Context` object, which is not interesting for the user.

Reply:
That's exactly what this PR does though.

Reply:
Sorry, I didn't explain myself very well. At the moment we are printing a nice representation of the routes.
Doing `JSON.stringify(router.tree, null, 2)` will show a huge amount of useless data for the user, see the `Context` object (in the router api is called `store`)

Reply:
Add a `toJSON()` method on `router.tree`.

Reply:
I agree that `printRoutes` would be a better name. Although the current implementation seems fine to me.